js/ui/metric-popup.js

```diff
@@ -194,6 +212,25 @@ export const MetricPopup = {
                 </select>
               </div>
             </div>
+            <table>
+              <tr>
+                <td>
+                  <label>
+                    <input type="checkbox" id='checkbox_min' checked=true v-on:click='changeMin'> min anzeigen
```

Comment:
Same here, the connection between the UI element and JS store should be using `v-model`.